src/vs/editor/contrib/stickyScroll/browser/stickyScroll.ts

```diff
@@ -335,6 +355,7 @@ class StickyScrollCodeLine {
 class StickyScrollWidget implements IOverlayWidget {
 
 	private readonly arrayOfCodeLines: StickyScrollCodeLine[] = [];
+	private readonly linesRange: number[] = [];
```

Comment:
Adding a member here is not necessary. Each `StickyScrollCodeLine` knows its line number, it just needs to expose a getter or make it public. See `StickyScrollCodeLine._lineNumber` which already has this.